app/templates/direct-award/index.html

```diff
@@ -52,16 +52,23 @@ <h1>
       empty_message="You have no saved searches",
       field_headings=[
         "Name",
-        "Ended"  ],
+        "Ended",
+        "Outcome"
+      ],
       field_headings_visible=True
     ) %}
       {% call summary.row() %}
-        {{ summary.service_link(item.name, url_for('direct_award.view_project', framework_family=framework.framework, project_id=item.id)) }}
+        {{ summary.service_link(item.name, url_for('direct_award.view_project', framework_family=framework.framework, project_id=item.id), wide=False) }}
         {{ summary.text(item.lockedAt | datetimeformat ) }}
+        {{ 
+          summary.text("Awarded") if item.outcome and item.outcome.result == "awarded"
+          else summary.text("The work has been cancelled") if item.outcome and item.outcome.result == "cancelled"
+          else summary.text("No suitable services found") if item.outcome and item.outcome.result == "none-suitable"
+          else summary.service_link("Tell us the outcome", url_for('direct_award.did_you_award_contract', framework_family=framework.framework, project_id=item.id), wide=False) }}
```

Comment:
Given this is an action, should the `Tell us the outcome` link be inside a `summary.field(action=True)` instead of a `summary.service_link`?

Reply:
@kubabartwicki ^^^

Reply:
from @kubabartwicki  - "i’ve played with some variants but this worked best"